slash-commands/slash-commands.js

```diff
@@ -156,13 +199,123 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 	/** Function: dnd
 	 * Change the current user's XMPP status to "dnd" with an optional message
 	 * Parameters:
-	 * 	(String) message Optional message to set with availability
+	 *    (String) message Optional message to set with availability
 	 */
 	self.dnd = function(message) {
 		// TODO: The message field is currently unsupported by Candy.Core.Action.Jabber.Presence
 		Candy.Core.Action.Jabber.Presence(null, $build('show', 'dnd'));
 	};
+	
+	/** Function: invite
+	 * invite another user to the current chat room, with lookups for real room & user jids
+	 *
+	 *Parameters:
+	 *    (String) user Nickname of user, or JID - all currently connected rooms will be checked
+	 *    (String) room Optional room name, must already exist
+	 *    (String) password Optional room password, if room requires one
+	 */
+	self.invite = function(args) {
+		if(args === undefined || args == ''){
+			Candy.View.Pane.Chat.onInfoMessage(self.currentRoom(), '', "usage: /invite user [room] [roomPassword] ");
+			return false;
+		}
+		args = args.split(' ');
```

Comment:
Same issue, when the user has a space in his nick; I won’t report it in other places, but it should be fixed globally.